near/src/runtime.rs

```diff
@@ -40,6 +40,7 @@ use near_store::{
 use node_runtime::adapter::ViewRuntimeAdapter;
 use node_runtime::state_viewer::TrieViewer;
 use node_runtime::{ApplyState, Runtime, StateRecord, ValidatorAccountsUpdate};
+use std::cmp::Ordering;
```

Comment:
ctrl + shift + O for optimize  imports
(Code | Optimize imports)

Reply:
`ctrl+shift+O` doesn't do the same than `rustfmt`, I have configured to `rustfmt` on save.